server/sonar-web/src/main/js/apps/organizations/components/forms/ManageMemberGroupsForm.js

```diff
@@ -104,6 +104,12 @@ export default class ManageMemberGroupsForm extends React.PureComponent {
     this.closeForm();
   };
 
+  handleClose = (e /*: MouseEvent */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)